### PR TITLE
Automatically port IO.Enumeration.FileSystemName remark

### PR DESCRIPTION
**Pull request by carlossanlop**
Area owners: @JeremyKuhne @carlossanlop

### CODE REVIEW

File: xml/System.IO.Enumeration/FileSystemName.xml [hunk 1]
```diff
@@ -87,7 +87,17 @@
         <summary> Verifies if the given Win32 expression matches the given name. Supports the following wildcards: '*', '?', '&lt;', '&gt;', '"'. The backslash character '\' escapes.</summary>
```
Comment (rpetrusha) [inline xml/System.IO.Enumeration/FileSystemName.xml]:
```suggestion
        <summary> Verifies if the given Win32 expression matches the given name. Supports the following wildcards: '*', '?', '&lt;', '&gt;', '"'. The backslash character `\` escapes.</summary>
```

File: xml/System.IO.Enumeration/FileSystemName.xml [hunk 2]
```diff
@@ -87,7 +87,17 @@
...
         <returns>
           <see langword="true" /> if the given expression matches the given name; otherwise, <see langword="false" />.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This is based off of <xref:System.IO.PatternMatcher> used in `FileSystemWatcher`, which is based off of `RtlIsNameInExpression`, which defines the rules for matching DOS wildcards (`'*'`, `'?'`, `'<'`, `'>'`, '"').
```
Comment (rpetrusha) [inline xml/System.IO.Enumeration/FileSystemName.xml]:
```suggestion
The syntax of the `expression` parameter is based on the syntax used by <xref:System.IO.FileSystemWatcher>, which is based on [RtlIsNameInExpression](/windows/win32/devnotes/rtlisnameinexpression), which defines the rules for matching DOS wildcards (`'*'`, `'?'`, `'<'`, `'>'`, '"').
```
PatternMatcher is an internal type, so the link is invalid.

File: xml/System.IO.Enumeration/FileSystemName.xml [hunk 3]
```diff
@@ -87,7 +87,17 @@
...
+ 
+Like `PatternMatcher`, matching will not line up with Win32 behavior unless you transform the expression using <xref:System.IO.Enumeration.FileSystemName.TranslateWin32Expression(System.String)>.
```
Comment (rpetrusha) [inline xml/System.IO.Enumeration/FileSystemName.xml]:
```suggestion
Matching will not correspond to Win32 behavior unless you transform the expression using <xref:System.IO.Enumeration.FileSystemName.TranslateWin32Expression(System.String)>.
```

File: xml/System.IO.Enumeration/FileSystemName.xml (hunk 1 above)
Comment (carlossanlop) [reply xml/System.IO.Enumeration/FileSystemName.xml]: This is not a remark, so I don't think the backticks will have an effect. I will only apply the space removal:
```suggestion
         <summary> Verifies if the given Win32 expression matches the given name. Supports the following wildcards: '*', '?', '&lt;', '&gt;', '"'. The backslash character '\' escapes.</summary>
```

File: xml/System.IO.Enumeration/FileSystemName.xml (hunk 2 above)
Comment (carlossanlop) [reply xml/System.IO.Enumeration/FileSystemName.xml]: Thanks for catching that. I will add a couple of missing backticks in the last wildcard:

```suggestion
The syntax of the `expression` parameter is based on the syntax used by <xref:System.IO.FileSystemWatcher>, which is based on [RtlIsNameInExpression](/windows/win32/devnotes/rtlisnameinexpression), which defines the rules for matching DOS wildcards (`'*'`, `'?'`, `'<'`, `'>'`, `'"'`).
```

**Comment by rpetrusha** [comment]
LGTM. Is this ready to merge, @carlossanlop?

**Comment by carlossanlop** [comment]
@rpetrusha Yes, looks good, thank you. Ready to merge.

**Pull request merged**